Tasks/Task002.playground/Contents.swift

```diff
@@ -8,7 +8,8 @@
 // MARK: - Solution
 //
 // Beginning of the solution
-
+var orderPrice: Double = 2000
```

Comment:
тут можно было опустить указание типа, swift бы привел его к `Double` самостоятельно
```suggestion
var orderPrice = 2000
```